IBMigrationTool/InterfaceBuilderFileMigrator.swift

```diff
@@ -50,6 +85,7 @@ final class InterfaceBuilderFileMigrator {
     
     /// Performs migration of the current file, storing the result in `string`
     func migrate() {
+        unmigratableMatches = [:]
```

Comment:
Could return from the method, or maybe make `unmigratableMatches` a `private(set)`

Reply:
Good shout, will return!